gulpfile.js

```diff
@@ -36,5 +37,11 @@ gulp.task('watch', ['build'], function() {
   gulp.watch(path.resolve(CSS, '**/*.styl'), ['css']);
 });
 
+// Fake API
+gulp.task('serve:api', function(done) {
+  cp.exec('node_modules/.bin/json-server --watch db.json --port 3009', {stdio: 'inherit'})
```

Comment:
Not a big deal, but we can optionally leave off the path (`node_modules/.bin/json-server` -> `json-server`) and Node will find the path on its own.